app/routes/reminders.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
```

Comment:
I'm leaving this here because I know you're about to go back and add functionality to display reminders, but in the future any file not being used should not be included in your PR.